integrations/gitlab/.port/resources/port-app-config.yaml

```diff
@@ -15,3 +15,21 @@ resources:
             readme: file://README.md
             description: .description
             language: .__languages | to_entries | max_by(.value) | .key
+  - kind: member
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          identifier: .username
+          title: .name
+          blueprint: '"member"'
+          properties:
+            state: .state
+            locked: .locked
+            link: .web_url
+            email: .email
+            publicEmail: .__public_email
+          relations:
+            gitlabGroup: '[.__groups[].full_path]'
+            createdBy: .created_by.username
```

Comment:
in the blueprint relation you only have gitlabGroup, while in your mapping you have both createdBy and gitlabGroup. I am not sure it is of interest for users who created the user, lets remove it. Let me know if you think otherwise